### ISSUE TITLE
Error Build with 1.0.1

### ISSUE THREAD

**Issue by FrancisCan** [mikepenz/MaterialDrawer#48]
I cannot build my project if update dependence from 1.0.0 to 1.0.1
```java
Error:A problem occurred configuring project ':app'.
> Could not resolve all dependencies for configuration ':app:_debugCompile'.
   > Could not find Android-Iconics:library:unspecified.
     Searched in the following locations:
         https://jcenter.bintray.com/Android-Iconics/library/unspecified/library-unspecified.pom
         https://jcenter.bintray.com/Android-Iconics/library/unspecified/library-unspecified.jar
         file:/C:/Android Sdk/sdk/extras/android/m2repository/Android-Iconics/library/unspecified/library-unspecified.pom
         file:/C:/Android Sdk/sdk/extras/android/m2repository/Android-Iconics/library/unspecified/library-unspecified.jar
         file:/C:/Android Sdk/sdk/extras/google/m2repository/Android-Iconics/library/unspecified/library-unspecified.pom
         file:/C:/Android Sdk/sdk/extras/google/m2repository/Android-Iconics/library/unspecified/library-unspecified.jar
     Required by:
         WebAnimex4:app:unspecified > com.mikepenz.iconics:google-material-typeface:1.1.1
```

And I can't understand why 
 ```java
Required by:
WebAnimex4:app:unspecified > com.mikepenz.iconics:google-material-typeface:1.1.1
```
I have    
```java
 compile 'com.mikepenz.iconics:google-material-typeface:1.1.0@aar'
```

**Comment by yoavst** [comment]
same here

**Comment by aimanbaharum** [comment]
Same here. I'm also using mavenCentral() in project's build.gradle.

**Comment by mikepenz** [comment]
Sorry something must have broken with the latest maven release

**Comment by mikepenz** [comment]
Fixed in v1.0.2
